### PR TITLE
interface: add firewall-control from ubuntu-core-security

### PR DESCRIPTION
**Pull request by jdstrand**

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by jdstrand** [comment]
I don't know why travis failed. this part of ./run-checks passes locally.

**Comment by jdstrand** [comment]
Apparently I was out of date. Fixed that and adjusted accordingly.

**Comment by zyga** [comment]
whitelist this please

File: interfaces/builtin/all.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 
 var allInterfaces = []interfaces.Interface{
 	&BoolFileInterface{},
+	NewFirewallControlInterface(),
```
Comment (zyga) [inline interfaces/builtin/all.go]: Please add a test for this to ``all_test.go``

**Comment by zyga** [comment]
One trivial change and LGTM.

**Pull request merged**